phpunit.xml.dist

```diff
@@ -6,6 +6,7 @@
         colors="true"
         displayDetailsOnTestsThatTriggerWarnings="true"
         displayDetailsOnTestsThatTriggerErrors="true"
+        cacheDirectory="var/phpunit/cache"
```

Comment:
@Bellangelo So the result cache file option was removed, but the result cache is always stored in the PHPUnit cache folder or in the root directory. Adding this option moves the result cache to the PHPUnit cache folder.